keyboards/baion_808/keymaps/default/readme.md

```diff
@@ -0,0 +1,3 @@
+# The default keymap for Baion 808. VIA support disabled.
+
+![Layer 0](https://i.imgur.com/IhbvSZ1.png)
```

Comment:
Both the keymap `readme.md`'s should be removed and this image displayed in keyboard level `readme.md`, if you feel necessary.